src/foam/dao/AbstractF3FileJournal.js

```diff
@@ -355,14 +363,17 @@ try {
           String line = reader.readLine();
           if ( line == null ) return null;
           if ( ! line.equals("p({") && ! line.equals("r({") ) return line;
-          StringBuilder sb = new StringBuilder();
-          sb.append(line);
+          strb.setLength(0);
```

Comment:
You have a thread local string builder - what is this one?

Reply:
Is used for reading, and is shared across threads, so threadlocal wouldn't work. Should add a comment and maybe give a more meaningful name.

Reply:
string builder is not thread local